### DISCUSSION TITLE
image.alpha argument missing from SpatialFeaturePlot

### DISCUSSION THREAD

**Discussion by dwmurray8** [satijalab/seurat#4682] [Q&A]
Hi everyone,

I want to remove the background image from my spatial feature plot, but it seems that can only be accomplished when creating a spatial plot. Why is this? Is there any way to remove the image when creating a spatial feature plot?

Thanks for any help that you can provide,
Duncan

**Comment by dwmurray8** [comment]
I was able to create the same type of plot using the SpatialPlot function, which did allow for the image.alpha argument.

**Comment by ggomes77** [comment]
With SpatialFeaturePlot you can use image = "False", this way only the spots are plotted.
